mews-api.md

```diff
@@ -464,9 +572,60 @@ This is example of a _successful_ response. In case an error occurred, the respo
 | `ratePlanCode` | `string` | required | Mapping code of the rate plan |
 | `spaceTypeCode` | `string` | required | Mapping code of the space type related to the rate plan. |
 
+#### Products
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `code` | `string` | required | Mapping code of the product. |
+| `name` | `string` | required | Name of the product. |
+| `description` | `string` | optional | Description of the product. |
+| `unitAmount` | `[`Amount object`](https://mews-systems.gitbook.io/channel-manager-api/mews-api#amount)` | required | A product cost. |
+| `currencyCode` | `string` | required | 3 letter currency code of the product. |
+| `netValue` | `decimal` | required | Tax exclusive product cost. |
+| `grossValue` | `decimal` | required | Tax inclusive product cost. |
+| `cancellationPolicies` | [`Cancellation Policy`](mews-api.md#cancellation-policy) collection | optional | Cancellation policies of the rate plan. |
+| `taxValues` | `object` | required | Identifies legal environment specific taxes. |
+| `code` | `string` | required | Tax code corresponding to legal environment. |
+| `value` | `decimal` | required | Tax amount. |
+| `pricing` | `string` | required | Identified in [`pricing type`](https://mews-systems.gitbook.io/channel-manager-api/mews-api#extra-pricing-types). |
+
+#### Product Mapping
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ratePlanCode` | `string` | required | Mapping code of the rate plan. |
+| `productCode` | `string` | required | Mapping code of the product related to the rate plan. |
+
+#### Taxations
```

Comment:
It looks like it shoudnt be here as its empty

Reply:
I just dont know all Taxation codes, but we receive questions on them

Reply:
https://develop.mews.li/Commander/Globalization/TaxEnvironments
and then maybe you are looking for sub codes

```Zero =  "Z"
   LowestReduced = "T"
   LowerReduced = "L"
   Reduced = "R"
   Standard = "S"
   Increased = "I"
   HigherIncreased = "H"
   HighestIncreased = "G"
   Maximum = "M"
```
and @allisonchihak  should know all these informations now. 😄